spec/mercure.md

```diff
@@ -447,29 +450,34 @@ response, however polyfills and server-sent events clients in most programming l
 The hub **CAN** also specify the reconnection time using the `retry` key, as specified in the
 server-sent events format.
 
-# Subscription Events {#subscriptions}
+# Subscription Events
+
+The hub **MAY** publish an update when a subscription is created or terminated. If this feature is
+implemented by the hub, an update **MUST** be dispatched every time that a subscription is created
+or terminated.
 
-The hub **MAY** publish an update when a subscription to a topic is created or terminated. If this
-feature is implemented by the hub, an update **MUST** be dispatched every time that a subscription
-is created or terminated, and for each topic to which the client subscribes.
+The topic of these updates **MUST** follow the pattern
+`/.well-known/mercure/subscriptions/{topicSelector}/{subscriberID}` where `{topic-selector}` is the
+URL-encoded topic selector used for this subscription and `{subscriberID}` is an unique identifier
+for the subscriber.
 
-The topic of this update **MUST** follow the pattern
-`/.well-known/mercure/subscriptions/{subscriptionID}/{topic}` where `subscriptionID` is an
-unique identifier for this subscription and `topic` is the URL-encoded value of the subscribed
-topic. `subscriptionID` **SHOULD** be an URL-encoded IRI [@!RFC3987]. An UUID [@RFC4122] or a DID
+If a subscriber has several subscriptions, it **MUST** be identified by the same identifier.
+`{subscriberID}` **SHOULD** be an URL-encoded IRI [@!RFC3987]. An UUID [@RFC4122] or a DID
 [@W3C.WD-did-core-20200421] **MAY** be used.
 
 The content of the update **MUST** be a JSON-LD [@!W3C.REC-json-ld-20140116] document containing at
 least the following properties:
 
- *  `@id`: the identifier of this update, it **MUST** be the same value as the main subscription
-    update's topic
+ *  `@id`: the identifier of this update, it **MUST** be the same value as the subscription update's
+    topic
 
  *  `@type`: the fixed value `https://mercure.rocks/Subscription`
 
- *  `topic`: the topic to which the subscription refers
+ *  `topicSelector`: the topic selector used of this subscription
```

Comment:
Should we use `topic` as the query parameter, or should we rename the `topic` query parameter to `topicSelector`? (Ex: `https://example.com/.well-known/mercure?topicSelector=https://example.com/foo/{bar}`)